scripts/pso-collect-logs.sh

```diff
@@ -0,0 +1,71 @@
+if [[ $# != 0 ]]; then
+  echo -e "If kubeconfig is not configured please run: export KUBECONFIG=[kube-config-file]\n"
+  echo -e "For openshift cluster, please change KUBECTL variable to oc\n"
+  echo -e "Usage: *.sh"
+  exit
+fi
+
+# Please change to oc for openshift cluster.
+KUBECTL=kubectl
```

Comment:
I think you could automate this... Example code:
```# Find out if this is OpenShift
OC=/usr/bin/oc
if [ -f "$OC" ]; then
  KUBECTL=oc
else
  KUBECTL=kubectl
fi```

Reply:
This might not always be the case, kubectl is run on any client and people may have both oc and kubectl and choose to connect to multiple clusters. for example in our dev environment (nsm-dev-tools) we always have both.

Reply:
@sdodsley what do you think, please take another look.

Reply:
@haibinxie It might be worth asking the question at the start of the script - OpenShift or not?
This will then make sure the correct binary is used. 
If they have both and do not remember to change the KUBECTL variable it could go badly...

Reply:
@sdodsley sounds good, resolved in a new commit.